kuberos.go

```diff
@@ -358,9 +365,28 @@ func populateUser(cfg *api.Config, p *extractor.OIDCAuthenticationParams) api.Co
 			},
 		},
 	}
+
 	for name, cluster := range cfg.Clusters {
+		// If the cluster definition does not come with certificate-authority-data nor
+		// certificate-authority, then check if kuberos has access to the cluster's CA
+		// certificate and include it when possible. Assume all errors are non-fatal.
+		if len(cluster.CertificateAuthorityData) == 0 && len(cluster.CertificateAuthority) == 0 {
```

Comment:
`cluster.CertificateAuthority` is a string, so it would be more idiomatic to test that it's the empty string (i.e. `cluster.CertificateAuthority == ""`.

Reply:
Good call! I'll make the change.